### PR TITLE
[WIP] Improve logging

### PR DESCRIPTION
**Pull request by smolck**
See #107.

EDIT: Haven't gone through the other files yet, so if you know of any files that need updating feel free to let me know.

### CODE REVIEW

File: src/ui/ui.rs [hunk 1]
```diff
@@ -391,7 +393,7 @@ fn handle_notify(
                 .cb(|res| match res {
                     Ok(_) => {}
                     Err(err) => {
-                        println!("Failed to execute nvim command: {}", err)
+                        error!("Failed to execute nvim command: {}", err)
```
Comment (smolck) [inline src/ui/ui.rs]: Should this line and the others like them be `error!`s instead of `debug!`s?

File: Cargo.toml [hunk 2]
```diff
@@ -20,7 +20,9 @@ libwebkit2gtk = [
 ]
 
 [dependencies]
-#neovim-lib = "0.6"
+log = "0.4.8"
+env_logger = "0.7.1"
```
Comment (smolck) [inline Cargo.toml]: I'm not 100% sure this crate is necessary.

File: src/nvim_bridge/mod.rs [hunk 3]
```diff
@@ -1,3 +1,5 @@
+use log::{debug, error};
```
Comment (smolck) [inline src/nvim_bridge/mod.rs]: Not sure if we want to change the positions of this and the other imports?

**Comment by vhakulinen** [comment]
Merged.

**Pull request merged**